frontend/package-lock.json

```diff
@@ -1,18474 +1,8 @@
 {
```

Comment:
You  probably did not mean to change our package-lock?

Reply:
I did not mean to change anything there.

Reply:
Resolved with 0d9cdab8155d4ba63ec44c40c11fb7dbdd808952